Extensions/Effects/zoom-blur-pixi-filter.js

```diff
@@ -0,0 +1,15 @@
+gdjs.PixiFiltersTools.registerFilterCreator('ZoomBlur', {
+  makePIXIFilter: function() {
+    var zoomBlur = new PIXI.filters.ZoomBlurFilter();
+
+    return zoomBlur;
+  },
+  updateParameter: function(filter, parameterName, value) {
+    if (!['centerX', 'centerY', 'strength', 'innerRadius'].includes(parameterName)) return;
```

Comment:
includes is unfortunately not supported by IE11. We must `.indexOf(parameterName) !== -1` instead (here and in the rest of the filters)

Reply:
I am refactoring this for all filters now. Sorry for the inconvenience :)